Framework/MDAlgorithms/src/LoadMD.cpp

```diff
@@ -650,10 +650,17 @@ CoordTransform *LoadMD::loadAffineMatrix(std::string entry_name) {
   m_file->getAttr<int>("rows", outD);
   m_file->getAttr<int>("columns", inD);
   m_file->closeData();
+  Matrix<coord_t> mat(outD, inD);
+  size_t ic(0);
+  for (size_t i = 0; i < static_cast<size_t>(outD); i++) {
+    for (size_t j = 0; j < static_cast<size_t>(inD); j++) {
+      mat[i][j] = vec[ic];
+      ic++;
+    }
+  }
```

Comment:
What do you think about updating or adding a constructor for Rectangular Matrices and using that instead?

http://doxygen.mantidproject.org/nightly/dc/da6/classMantid_1_1Kernel_1_1Matrix.html#a265deafa95ceb2e94950b127f45df08c